content/200-concepts/100-components/02-prisma-client/056-composite-types.mdx

```diff
@@ -21,6 +15,8 @@ Composite types are only available with MongoDB.
 
 [Composite types](/concepts/components/prisma-schema/data-model#defining-composite-types), known as [embedded documents](https://docs.mongodb.com/manual/core/data-model-design/#std-label-data-modeling-embedding) in MongoDB, allow you to embed records within other records.
 
+We made composite types [Generally Available](/about/prisma/releases#generally-available-ga) in v3.12.0. They were previously available in [preview](concepts/components/preview-features) from v3.10.0.
```

Comment:
```suggestion
We made composite types [Generally Available](/about/prisma/releases#generally-available-ga) in v3.12.0. They were previously available in [Preview](concepts/components/preview-features) from v3.10.0.
```

Not sure about the capitalization here, I feel like if Generally Available is capitalized so should preview?

Reply:
However, you're right that this jumps out in this paragraph, so I'll capitalize preview for now.